Formula/ffmpeg.rb

```diff
@@ -1,16 +1,43 @@
 class Ffmpeg < Formula
   desc "Play, record, convert, and stream audio and video"
   homepage "https://ffmpeg.org/"
-  url "https://ffmpeg.org/releases/ffmpeg-3.1.4.tar.bz2"
-  sha256 "7c99df75a4dc12d22c0f1ed11d0acf98cac1f8b5fe7a7434344b167f810bcbfa"
-  head "https://github.com/FFmpeg/FFmpeg.git"
+
+  stable do
+    url "https://ffmpeg.org/releases/ffmpeg-3.1.4.tar.bz2"
+    sha256 "7c99df75a4dc12d22c0f1ed11d0acf98cac1f8b5fe7a7434344b167f810bcbfa"
+
+    option "with-ffplay", "Enable FFplay media player"
+    option "with-openh264", "Enable OpenH264 library (v1.5.0)"
+
+    depends_on "sdl" if build.with? "ffplay"
```

Comment:
This is referring to the `stable` branch, in this case `v3.1.4`, and it is in FFmpeg `version <next>` that had support for `SDL2` added and for `SDL1` removed. (which is the `head` version of this formula)